src/index.js

```diff
@@ -90,8 +91,8 @@ class RichMarkdownEditor extends React.PureComponent<Props, State> {
     if (this.props.readOnly) return;
     window.addEventListener("keydown", this.handleKeyDown);
 
-    if (!this.props.defaultValue) {
-      this.focusAtStart();
+    if (this.props.autoFocus) {
+      this.focusAtEnd();
```

Comment:
Ah man, I missed that you changed this from `focusAtStart` to `focusAtEnd` ... I'm going to change this in 5.0.1 to focus at the start if `autoFocus && !this.props.defaultValue` otherwise it really screws up Outline which has an empty title and paragraph by default

cc @jorilallo